pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -364,18 +367,32 @@ protected long getCurrentCountStarResult()
    * @param timeoutMs Timeout in milliseconds
    * @throws Exception
    */
-  protected void waitForAllDocsLoaded(long timeoutMs)
-      throws Exception {
+  protected void waitForAllDocsLoaded(long timeoutMs) throws Exception {
     final long countStarResult = getCountStarResult();
     TestUtils.waitForCondition(new Function<Void, Boolean>() {
       @Nullable
       @Override
       public Boolean apply(@Nullable Void aVoid) {
+        long currentResult;
         try {
-          return getCurrentCountStarResult() == countStarResult;
+          currentResult = getCurrentCountStarResult();
         } catch (Exception e) {
           return null;
         }
+
+        // Add more information to logs
+        if (currentResult == countStarResult) {
+          return true;
+        } else if (currentResult < countStarResult) {
+          LOGGER.info("Docs are not fully loaded yet (current: {}, expected: {})", currentResult, countStarResult);
```

Comment:
we don't want this message printed every 100ms. I think we should print it every 1/10th of the total time (so, every minute if we are waiting for 10 minutes) or every second, whichever is larger.

Can you change the waitForCondition accodingly?

If we do this, we can also print more stuff (e.g. which segments are loaded)